CSharp/Library/Microsoft.Bot.Connector.NetFramework/BotAuthentication.cs

```diff
@@ -77,7 +77,12 @@ public override async Task OnActionExecutingAsync(HttpActionContext actionContex
             await base.OnActionExecutingAsync(actionContext, cancellationToken);
         }
 
-        private IList<Activity> GetActivities(HttpActionContext actionContext)
+        protected async Task BaseOnActionExecutingAsync(HttpActionContext actionContext, CancellationToken cancellationToken)
```

Comment:
why do we need this? is base.OnActionExecutingAsync private?

Reply:
Need to call ActionFilterAttribute.OnActionExecutingAsync from SkillBotAuthentication.  Is there a better way to skip the middle inheriting class method (c# doesn't allow base.base.method()) ?  SkillBotAuthentication -> BotAuthentication -> ActionFilterAttribute

Reply:
Changed BaseOnActionExecutingAsync to ContinueOnActionExecutingAsync